src/main/java/me/zeroeightsix/kami/feature/module/player/Blink.java

```diff
@@ -62,9 +67,14 @@ public void onEnable() {
 
     @Override
     public void onDisable() {
-        if (packets.size() > maxPacketAmount && packetDiscard) {
-            if (clonedPlayer != null) // We don't want a NPE when a player tries to disable blink after logging in
-                mc.player.setPos(clonedPlayer.getPos().x, clonedPlayer.getPos().y, clonedPlayer.getPos().z); // Snap back to where we were
+        if (packets.size() > maxPacketAmount &&
+                isCancelable && clonedPlayer != null &&
+                clonedPlayer.getPos().distanceTo(mc.player.getPos()) > 10) { // If its more than 10 blocks
+
+            System.out.println("Based and Redpilled");
```

Comment:
Can you remove this line?

Reply:
Sure, that was a debug line I just forgot to get rid of.